webhook_server/libs/github_api.py

```
@@ -472,14 +472,23 @@ async def process(self) -> Any:
 
             if await asyncio.to_thread(lambda: pull_request.draft):
-                self.logger.debug(f"{self.log_prefix} Pull request is draft, doing nothing")
-                token_metrics = await self._get_token_metrics()
-                self.logger.info(
-                    f"{self.log_prefix} Webhook processing completed successfully: "
-                    f"draft PR (skipped) - {token_metrics}",
+                allow_commands_on_draft = self.config.get_value("allow-commands-on-draft-prs")
+
+                # Only allow issue_comment events when config is explicitly set
+                if allow_commands_on_draft is None or self.github_event != "issue_comment":
+                    self.logger.debug(f"{self.log_prefix} Pull request is draft, doing nothing")
```

Ruff (0.14.13)

Logging statement uses f-string

(G004)